releasing/plugin-commands-publishing/src/publish.ts

```diff
@@ -205,6 +205,8 @@ Do you want to continue?`,
       args.splice(index, 2)
     }
   }
+  // Convert Pnpm's "unknown config" form into regular form so that npm can understand it
+  args = args.map(a => a.replace(/^--config\./, '--'))
```

Comment:
Note: I just realized that we need to make sure we should not apply this conversion after `--`. I'll fix later after getting some signal from maintainers